src/lib/construction.ts

```diff
@@ -0,0 +1,182 @@
+import { Field, Proof } from '../snarky';
```

Comment:
this file is not used anywhere in the code - delete it? or needed later? @imeckler